src/log.ts

```diff
@@ -5,22 +5,33 @@ import { visibility } from './onVisibilityChange';
 import { po } from './performanceObserver';
 import { reportPerf } from './reportPerf';
 import { initTotalBlockingTime } from './totalBlockingTime';
-import { Metric } from './types';
+import { IPerfumeData, Metric } from './types';
 import { roundByFour } from './utils';
 import { getVitalsScore } from './vitalsScore';
 
 export const logData = (
   measureName: string,
-  metric: any,
+  metric: IPerfumeData,
   attribution?: object,
 ): void => {
-  Object.keys(metric).forEach(key => {
-    if (typeof metric[key] === 'number') {
-      metric[key] = roundByFour(metric[key]);
+  const roundNumericProperties = (data: IPerfumeData): IPerfumeData => {
+    if (typeof data === 'number') {
```

Comment:
Note that this if statement is designed to replicate the original behavior, if I get it right. It's unclear for why `IPerfumeData` values of type `number` are not rounded, unlike objects.